scripts/dxcompiler_copy/dxapp.json

```diff
@@ -99,10 +99,17 @@
           "instanceType": "mem1_ssd1_v2_x2"
         }
       }
+    },
+    "azure:uksouth-ofh": {
+      "systemRequirements": {
+        "*": {
+          "instanceType": "azure:mem2_ssd1_x2"
+        }
+      }
     }
   },
   "details": {
-    "whatsNew": "* 0.0.2: Added new London region."
+    "whatsNew": "* 0.0.3: Added new London OFH region * 0.0.2: Added new London region."
```

Comment:
```suggestion
    "whatsNew": "* 0.0.3: Added new London OFH region\n* 0.0.2: Added new London region."
```

Reply:
fixed in [dd4730a](https://github.com/dnanexus/dxCompiler/pull/407/commits/dd4730a369b0c731832d98415aa7a5be7e02e0b2)